source/databases-collections.txt

```diff
@@ -117,33 +117,32 @@ Create a New Database
 When you create a new database, you must populate it with an
 initial collection.
 
-To create a new database:
-
-1. Hover over the connection for the deployment where you want your
+#. Hover over the connection for the deployment where you want your
```

Comment:
Small thing, feel free to disregard:
A user can also right click to open the context menu and choose to create a new database or collection from their instead of the hovering button if they want. Should we mention that here as well?
<img width="277" alt="Screenshot 2023-07-24 at 1 45 52 PM" src="https://github.com/mongodb/docs-mongodb-vscode/assets/1791149/7fa84b84-fa74-4c23-b406-3e63feb418ba">

Reply:
Thanks I added this for additional context.